packages/modules/imodel-browser/src/containers/iModelGrid/useIModelData.ts

```diff
@@ -21,7 +21,7 @@ export interface IModelDataHookOptions {
   apiOverrides?: ApiOverrides<IModelFull[]>;
   searchText?: string | undefined;
   maxCount?: number;
-  viewMode?: ViewType;
+  viewMode?: ViewType /** @deprecated in 2.1 It is no longer used as it has no effect on the data fetching. */;
```

Comment:
the comment here might need to move above the property. not sure if jsdoc will pick this up if its on the same line, and even if it does, not typically what we encourage